recipe/meta.yaml

```diff
@@ -1,18 +1,13 @@
-{% set version = "5.0.0" %}
+{% set version = "5.1.0" %}
 
 package:
   name: pyface
   version: {{ version }}
 
 source:
   fn: pyface-{{ version }}.tar.bz2
-  url: https://pypi.python.org/packages/source/p/pyface/pyface-{{ version }}.tar.bz2
-  md5: 0468d6a44adba0b70ec5171d1af9350a
-  patches:
-    # fix for issue https://github.com/enthought/pyface#188 following the
-    # solution from https://github.com/enthought/pyface/commit/06d85f8
-    # and raise syntax fix from https://github.com/enthought/pyface/pull/211
-    - pyface_fix.patch
+  url: https://pypi.python.org/packages/00/ec/04b1d7f1981107cde01bbc4a53ae2a234493e694cfa880fc00817e6c2a42/pyface-{{ version }}.tar.bz2
```

Comment:
Personally, yes, I prefer to go to the source, rather than pypi. It isn't hard and fast though - sometimes PyPI *is* the preferred source when the sdist isn't the same as the tagged ```.tar.gz```.

In this case though, sounds like they are like for like, and I'd lean towards the GH URL.